package.json

```diff
@@ -19,7 +19,7 @@
   "license": "BSD-2-Clause",
   "devDependencies": {
     "babel-code-frame": "^6.22.0",
-    "babylon": "^7.0.0-beta.19",
+    "babylon": "^7.0.0-beta.20",
```

Comment:
FYI `7.0.0-beta.20` contained an important fix for exported TS constructs so that is why I have set that as the new baseline